bundles/persistence/org.openhab.persistence.rrd4j/README.md

```diff
@@ -1,123 +1,183 @@
+
+Defines the type of data to be stored.
+It must be one of the following string values:
 
 - **COUNTER** represents a ever-incrementing value (historically this was used for packet counters or traffic counters on network interfaces, a typical home-automation application would be your electricity meter). If you store the values of this counter in a simple database and make a chart of that, you'll most likely see a nearly flat line, because the increments per time are small compared to the absolute value (e.g. your electricity meter reads 60567 kWh, and you add 0.5 kWh per hour, than your chart over the whole day will show 60567 at the start and 60579 at the end of your chart. That is nearly invisible. RRD4J helps you out and will display the difference from one stored value to the other (depending on the selected size). Please note that the persistence extensions will return difference instead of the actual values if you use this type; this especially leads to wrong values if you try to restoreOnStartup!
 - **GAUGE** represents the reading of e.g. a temperature sensor. You'll see only small deviation over the day and your values will be within a small range, clearly visible within a chart.
 - **ABSOLUTE** is like a counter, but RRD4J assumes that the counter is reset when the value is read. So these are basically the delta values between the reads.
 - **DERIVE** is like a counter, but it can also decrease and therefore have a negative delta.
 
-### Heartbeat, MIN, MAX
+### `<heartBeat>` (Heart Beat)
+
+The heartbeat parameter helps the database to detect missing values.
+i.e. if no new value is stored after "heartBeat" seconds, the value is considered missing when charting.
+
+It must be a positive integer value.
+
+### `<minValue> / <maxValue>` (Minimum resp. Maximum Value)
+
+These parameters define the range of acceptable sample values for that datasource.
+They must be either:
+
+- A numeric value, or
+- The letter "U" (unlimited)
+
+### `<sampleInterval>` (Sample Interval)
+
+The time interval (seconds) between reading consecutive samples from the OpenHAB core.
+
+It must be a positive integer value.
+
+### `<consolidationFunction>` (Consolidation Function)
+
+Determines the type of data compression to be used when more than one sample is to be stored in a single "storage box".
+So if you use (say) the `AVERAGE` function, and two samples of `20.0` and `21.0` are to be stored, then the value `20.5` would be stored in the box.
 
-Each datasource also has a value for heartbeat, minimum and maximum. This heartbeat setting helps the database to detect "missing" values, i.e. if no new value is stored after "heartbeat" seconds, the value is considered missing when charting. Minimum and maximum define the range of acceptable values for that datasource.
+It must be one of the following strings:
 
-### Step(s)
+- **AVERAGE** the average of all the samples is stored in the box
+- **MIN** the highest sample is stored in the box
```

Comment:
Don't you have MIN and MAX reversed here?

Reply:
Oops!